bin/ci/script.sh

```diff
@@ -15,7 +15,7 @@ if [[ "$ORCA_JOB" ]]; then
   eval "orca ci:run $ORCA_JOB script $ORCA_SUT_NAME"
 fi
 
-if [[ "$ORCA_ENABLE_NIGHTWATCH" == "TRUE" && "$ORCA_SUT_HAS_NIGHTWATCH_TESTS" && -d "$ORCA_YARN_DIR" ]]; then
+if [[ "$ORCA_ENABLE_NIGHTWATCH" == "TRUE" && "$ORCA_SUT_HAS_NIGHTWATCH_TESTS" && -d "$ORCA_YARN_DIR" && "$ORCA_SUT_NAME" != "drupal/example" ]]; then
```

Comment:
We still have handling for an internal-only (self-test) variable here--`$ORCA_SUT_NAME`--which is what we're trying to avoid. You should remove it from this condition here and then set `$ORCA_ENABLE_NIGHTWATCH` to `FALSE` before running `bin/ci/script.sh` in your self-test CI configuration, e.g., GitHub Actions. Then set it back to `TRUE` before running `bin/ci/self-test/script.sh`. That way it will be disabled in the common script that customers use, without having to be handled in customer-facing code. But it will still work for us.

Reply:
Done